Makefile

```diff
@@ -38,12 +38,30 @@ lint: lint.sub
 push: push.sub
 test: test.sub
 
+clean:
```

Comment:
interesting, I must have done something wrong in my playing with this. Thanks for the test!
Either way - i liked your other suggestion better so I went with that :-)